apps/prairielearn/src/tests/groupExam.test.ts

```diff
@@ -412,19 +452,25 @@ describe('cross exam assessment access', { timeout: 20_000 }, function () {
 
   it("prevents unauthorized users from accessing other groups' assessment instances", async function () {
     // Get exam assessment URL using ids from database
-    const firstAssessmentResult = await queryOneRowAsync(sql.select_group_exam_by_tid, {
-      assessment_tid: GROUP_EXAM_1_TID,
-    });
-    assert.lengthOf(firstAssessmentResult.rows, 1);
-    const firstAssessmentId = firstAssessmentResult.rows[0].id;
+    const firstAssessmentResult = await queryRow(
+      sql.select_group_exam_by_tid,
+      {
+        assessment_tid: GROUP_EXAM_1_TID,
+      },
```

Comment:
Prettier [does not simplify this by itself](https://prettier.io/docs/rationale#multi-line-objects):
```suggestion
      { assessment_tid: GROUP_EXAM_1_TID },
```